recipes/itolapi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "itolapi" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 161759384603664ff21ad6ae412a01549ce19436fc6959478d932d9b6c9915c4
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python<3.8
+    - setuptools
+    - requests >=2.0,<3.0
+  run:
+    - python<3.8
+    - requests >=2.0,<3.0
+
+test:
+  imports:
+    - itolapi
+    - itolapi.tests
+  requires:
+    - codecov >=2.0.9,<3.0.0
+    - coverage >=4.5.0,<5.0.0
+    - mock >=3.0.0,<4.0.0
+
+about:
+  home: https://github.com/albertyw/itolapi
+  license: MIT License
+  license_family: MIT
+  license_file: ''
+  summary: API for interacting with itol.embl.de
+  description: "iTOL Python API\n===============\n\n|PyPI|\n|Python Version|\n|License|\n\n|Codeship|\n|CodeCov|\n|Code Climate|\n|Dependency Status|\n\n\nPython API for the Interactive Tree of Life (iTOL)\n\
```

Comment:
This will not render correctly. Can you simplify this? Most other recipes just use plain text here. (Did you use a tool to create the recipe?)

Reply:
I used **conda skeleton** to create the recipe indeed. I have simplified the description